easybuild/easyconfigs/f/fastQValidator/fastQValidator-0.1.1a-dev-goolf-1.7.20-aadc6f9.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'fastQValidator'
+version = '0.1.1a-dev'
```

Comment:
I would go with `0.1.1a-20151214` (i.e. the date of the commit), since that provides more information